ci-operator/templates/openshift/openshift-ansible/cluster-scaleup-e2e-40.yaml

```diff
@@ -130,6 +130,10 @@ objects:
 
         # wait for the API to come up
         while true; do
+          if [[ -f /tmp/shared/scaleup-failed ]]; then
+            echo "Scaleup reported a failure, do not report test failure" 2>&1
+            exit 0
```

Comment:
At https://github.com/openshift/release/pull/7244/files#diff-91753b0515d25fa2f00c51d97df2605fR137 we `exit 1` if setup fails so it seems like this should be `exit 1` as well.

Reply:
I'm following the same pattern as in #6654.  I think the intent here is that if scaleup (or previously setup) fails, that the test job will not also report failed (exit 1) because the tests actually never ran.  The exit 1 on /tmp/shared/exit is a catch-all.  @smarterclayton can correct me if I'm wrong here.

Reply:
/lgtm